<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemDefValueForm</class>
 <widget class="QWidget" name="ItemDefValueForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Num. required values</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leNumReqValues"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Max. num. values</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leMaxNumValues"/>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="cbExtensible">
       <property name="text">
        <string>Extensible</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Common value label</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leCommonLabel"/>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="cbCommonLabel">
       <property name="text">
        <string>Use common label</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Units</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="leUnits"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="wSubclass" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
